www/js/diary/services.js

```diff
@@ -85,12 +85,12 @@ angular.module('emission.main.diary.services', ['emission.services'])
         'text-align': 'center',
         'background-color': '#ff5251',
         'color': 'black',
-        'width': '48px',
+        'width': '18%',
         'height': '16px',
```

Comment:
I don't have a strong opinion. Was just commenting on the general px v/s em v/s % debate.